perception/object_merger/src/object_association_merger/node.cpp

```diff
@@ -152,7 +153,7 @@ void ObjectAssociationMergerNode::objectsCallback(
       for (const auto & known_object : known_objects) {
         if (isUnknownObjectOverlapped(
               unknown_object, known_object, overlapped_judge_param_.precision_threshold,
-              overlapped_judge_param_.recall_threshold)) {
+              overlapped_judge_param_.recall_threshold, distance_threshold_)) {
```

Comment:
Can you add the overlapped_judge_param_ struct?
sample: `overlapped_judge_param_.distance_threshold;`

Reply:
I added distance_threshold_list to  overlapped_judge_param_ struct.